### DISCUSSION TITLE
错误：绕过代理服务器地址设置失败，应该怎么处理？

### DISCUSSION THREAD

**Discussion by liumingan** [vernesong/OpenClash#3855] [Q&A]
错误：绕过代理服务器地址设置失败，【undefined method `key?' for "hysteria://109.104.152.194:58510?upmbps=11&downmbps=55&auth=dongtaiwang.com&insecure=1&peer=bing.com&alpn=h3

**Comment by vernesong** [comment]
要改成yaml格式才能读取

**Reply by liumingan** [reply]
是yaml格式呀

**Reply by vernesong** [reply]
直接写的这种服务器信息格式是读取不了的，你忽略错误即可
